src/main.c

```diff
@@ -443,1273 +451,1609 @@ typedef struct {
 #endif
-static float se_dpi_scale(){
-  if(gui_state.dpi_override)return gui_state.dpi_override/120.;
+static float se_dpi_scale() {
+  if (gui_state.dpi_override)
+    return gui_state.dpi_override / 120.;
   static float dpi_scale = -1.0;
-  if(dpi_scale>0.)return dpi_scale;
+  if (dpi_scale > 0.)
+    return dpi_scale;
   dpi_scale = sapp_dpi_scale();
-  if(dpi_scale<=0)dpi_scale=1.;
-  dpi_scale*=1.10;
+  if (dpi_scale <= 0)
+    dpi_scale = 1.;
+  dpi_scale *= 1.10;
 #ifdef PLATFORM_ANDROID
   dpi_scale = se_android_get_display_dpi_scale();
 #endif
   return dpi_scale;
 }
 
-static void se_cache_glyphs(const char* input_string){
-  #ifdef UNICODE_GUI
-  utf8proc_int32_t codepoint_ref=0;
+static void se_cache_glyphs(const char *input_string) {
+#ifdef UNICODE_GUI
+  utf8proc_int32_t codepoint_ref = 0;
   const utf8proc_uint8_t *str = (const utf8proc_uint8_t *)input_string;
-  while(str[0]){
+  while (str[0]) {
     int size = utf8proc_iterate(str, -1, &codepoint_ref);
-    if(size<=0)break;
-    str+=size;
-    if(codepoint_ref>SE_MAX_UNICODE_CODE_POINT)continue;;
-    uint32_t font_cache_page = codepoint_ref/SE_FONT_CACHE_PAGE_SIZE;
-    if(gui_state.font_cache_page_valid[font_cache_page]==0x0){
-      gui_state.font_cache_page_valid[font_cache_page]=0x1;
-      gui_state.update_font_atlas=true;
-    }
-  }
-  #endif
-}
-char* se_replace_fake_path(char * new_path){
-  static char fake_path[SB_FILE_PATH_SIZE]; 
-  if(gui_state.fake_paths){
-    const char* base, *filename, *ext; 
-    sb_breakup_path(new_path,&base, &filename,&ext);
-    char* new_base = "/fakepath/";
-    if(gui_state.ui_type==SE_UI_ANDROID)new_base = "/storage/emulated/0/Android/data/com.sky.SkyEmu/";
-    snprintf(fake_path,sizeof(fake_path),"%s%s.%s",new_base,filename,ext);
-    new_path = fake_path; 
+    if (size <= 0)
+      break;
+    str += size;
+    if (codepoint_ref > SE_MAX_UNICODE_CODE_POINT)
+      continue;
+    ;
+    uint32_t font_cache_page = codepoint_ref / SE_FONT_CACHE_PAGE_SIZE;
+    if (gui_state.font_cache_page_valid[font_cache_page] == 0x0) {
+      gui_state.font_cache_page_valid[font_cache_page] = 0x1;
+      gui_state.update_font_atlas = true;
+    }
+  }
+#endif
+}
+char *se_replace_fake_path(char *new_path) {
+  static char fake_path[SB_FILE_PATH_SIZE];
+  if (gui_state.fake_paths) {
+    const char *base, *filename, *ext;
+    sb_breakup_path(new_path, &base, &filename, &ext);
+    char *new_base = "/fakepath/";
+    if (gui_state.ui_type == SE_UI_ANDROID)
+      new_base = "/storage/emulated/0/Android/data/com.sky.SkyEmu/";
+    snprintf(fake_path, sizeof(fake_path), "%s%s.%s", new_base, filename, ext);
+    new_path = fake_path;
   }
   return new_path;
 }
-static inline const char* se_localize_and_cache(const char* input_str){
-  const char * localized_string = se_localize(input_str);
+static inline const char *se_localize_and_cache(const char *input_str) {
+  const char *localized_string = se_localize(input_str);
```

Comment:
Can you configure the formatting to not touch the location of * in variable names?

Reply:
dunno but I will try

Reply:
No sadly, it does not seem like it, but we can tell it to put pointers to the left, right or in the middle. Maybe I simply missed it in the docs because it does feel like it should be an option but it is not.

Reply:
Does it still format them even if you don't have a `PointerAlignment` field?

Reply:
I can look into it.

Reply:
Nope it defaults to right alignment if I do that.